index.html

```diff
@@ -12,34 +13,42 @@
 
   <body>
     <header>
-      <section>
-        <h1>Bronwyn Detlofsen</h1>
-        <h5>Web Developer</h5>
-      </section>
-      <nav>
-        <ul id="navigation">
-          <li class="nav" data-category="skills"><a href="#">Skills</a></li>
-          <li class="nav" data-category="projects"><a href="#">Projects</a></li>
-          <li class="nav" data-category="experience"><a href="#">Experience</a></li>
-          <li class="nav" data-category="interests"><a href="#">Interests</a></li>
-          <li class="nav" data-category="profile"><a href="#">Profile</a></li>
-        </ul>
-      </nav>
+      <div>
+        <section>
+          <h1>Bronwyn Detlofsen</h1>
+          <h5>Web Developer</h5>
+        </section>
+        <nav>
+          <ul id="navigation">
+            <a href="#"><li class="nav" data-category="skills">Skills</li></a>
+            <li class="nav" data-category="projects"><a href="#">Projects</a></li>
+            <li class="nav" data-category="experience"><a href="#">Experience</a></li>
+            <li class="nav" data-category="interests"><a href="#">Interests</a></li>
+            <li class="nav" data-category="profile" id="lastli"><a href="#">Profile</a></li>
```

Comment:
Might be easier to use a pseudo-selector of **li:last-child** to target that final li rather than giving it its own id? Perhaps in case you added a navigation li later after it?